test/data/align_1.ref.aligned.nlp

```diff
@@ -1,13 +1,13 @@
-token|speaker|ts|endTs|punctuation|case|tags|wer_tags|oldTs|oldEndTs|ali_comment
-a|1|1.0000|2.0000||CA|[]|[]|||
-b|1|3.0000|4.0000||LC|[]|[]|||
-c|1|5.0000|6.0000||LC|[]|[]|||
-d|1|7.0000|8.0000|,|LC|[]|[]|||
-<laugh>|1|9.0000|10.0000|.|LC|['0:FALLBACK']|[]|||sub(<unk>)
-e|1|11.0000|12.0000||LC|[]|[]|||
-f|1|13.0000|14.0000||LC|[]|[]|||
-g|1|15.0000|16.0000||LC|[]|[]|||
-h|1|17.0000|18.0000||LC|[]|[]|||
-<foreign>|1|||,|LC|[]|[]|||del
-i|1|21.0000|22.0000||LC|[]|[]|||
-j|1|23.0000|24.0000||LC|[]|[]|||
+token|speaker|ts|endTs|punctuation|prepunctuation|case|tags|wer_tags|oldTs|oldEndTs|ali_comment
```

Comment:
To be safe, consider adding prepunctuation to one of these tests to make sure it gets stitched back correctly